webviz_subsurface/_providers/ensemble_table_provider/_field_metadata.py

```diff
@@ -0,0 +1,24 @@
+from typing import Optional
+
+import pyarrow as pa
+
+from .ensemble_table_provider import TableVectorMetadata
+
+
+def create_vector_metadata_from_field_meta(
```

Comment:
Currently, the `unit` property in `TableVectorMetadata` is defined as being optional, which implies that it is legal to return an empty `TableVectorMetadata` object. This function will however return `None` if metadata *is* present, but the `unit` key is missing.
If the current behavior of this function is what we want, the `unit` field in `TableVectorMetadata` should not be optional.
Alternatively, if the `unit` field should be kept as optional in `TableVectorMetadata`, this function should return an empty `TableVectorMetadata` object instead of returning `None` on line 20.

Reply:
I see. My thinking behind making it `Optional` was to keep it flexible, in case a future use of `EnsembleTableProvider` has column metadata that is not `unit`. But then it is better to return an empty `ColumnMetadata` as you say.